locations/spiders/toom_baumarkt_de.py

```diff
@@ -0,0 +1,10 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class ToomBaumarktDeSpider(SitemapSpider, StructuredDataSpider):
+    name = "toom_baumarkt_de"
+    item_attributes = {"brand": "Toom Baumarkt", "brand_wikidata": "Q2442970"}
+    sitemap_urls = ["https://static.toom.de/sitemap/cms/newmarkets-0.xml"]
```

Comment:
```suggestion
    sitemap_urls = ["https://static.toom.de/sitemap/cms.xml"]
    sitemap_follow = ["newmarkets"]
```

This will mean we'll get `newmarkets-1.xml` automatically if it ever gets added.